sql/updates/world/3.3.5/9999_99_99_99_world.sql

```diff
@@ -0,0 +1,11 @@
+-- repeal/replace some wrong spells
+DELETE FROM `spell_script_names` WHERE `ScriptName` IN ('spell_gen_despawn_self', 'spell_deliver_stolen_horse');
+INSERT INTO `spell_script_names` (`spell_id`, `ScriptName`) VALUES
+(15998, 'spell_gen_despawn_self'),
+(25952, 'spell_gen_despawn_self'),
+(29435, 'spell_gen_despawn_self'),
+(45980, 'spell_gen_despawn_self'),
+(51592, 'spell_gen_despawn_self'),
+(51910, 'spell_gen_despawn_self'),
+(52264, 'spell_deliver_stolen_horse'),
+(54420, 'spell_gen_despawn_self');
```

Comment:
don't delete all these spell_gen_despawn_self and readd them again, just delete what is needed

Reply:
So in this case, an exception can be made from the rule of "only delete entries in `spell_script_names` by `ScriptName` " -- so we only delete + insert `52267  spell_gen_despawn_self` by `spell_id` in addition to regular handling of `spell_deliver_stolen_horse` ?